discovery/discovery.go

```diff
@@ -40,8 +50,14 @@ func NewOrchestratorPool(node *core.LivepeerNode, addresses []string) *orchestra
 		glog.Error("Could not parse orchAddresses given - no URIs returned ")
 	}
 
+	var randomizedUris []*url.URL
+	r := rand.New(rand.NewSource(time.Now().Unix()))
```

Comment:
Let's remove this and just use the default state instead, eg `rand.Perm` . The LivepeerNode initializes the state at startup, and we can control it during testing.